server/scripts/seed.ts

```diff
@@ -16,6 +16,69 @@ import { RequestType } from '../models/requestTypeModel'
 
 dotenv.config()
 
+const numClients = 50
+const numGroups = 5
+const numTypesPerGroup = 5
+const numRequestsPerType = 50
+
+faker.seed(2021)
+
+const createSavePromise = (dbObject, msg) => {
+  const promise = dbObject.save().catch((err) => {
+    if (err) {
+      console.error('\x1b[31m', msg)
+      console.log('\x1b[0m')
+      console.error(err)
+      exit()
+    }
+  })
+  return promise
+}
+
+const createClient = (clientIDs, errMsg) => {
+  const client = new Client({
+    _id: mongoose.Types.ObjectId(),
+    clientId: faker.random.alphaNumeric(8),
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName()
+  })
+  clientIDs.push(client._id) // store IDs to allocate among groups + types
+  return createSavePromise(client, errMsg)
+}
+
+const createRequest = (requestID, clientIDs, requestTypeID, errMsg) => {
+  const request = new Request({
+    _id: requestID,
+    requestId: faker.random.alphaNumeric(6),
+    client: faker.random.arrayElement(clientIDs),
+  })
+  return createSavePromise(request, errMsg)
+}
+
+const createRequestType = (typeID, requestIDsForType, requestGroupID, errMsg) => {
+  const type = new RequestType({
+    _id: typeID,
+    name: faker.commerce.product(),
+    requests: {
+      fulfilled: [],
+      deleted: [],
+      open: requestIDsForType
+    },
+  })
+  return createSavePromise(type, errMsg)
+}
+
+const createRequestGroup = (groupID, typeIDs, errMsg) => {
+  const group = new RequestGroup({
+    _id: groupID,
+    name: faker.commerce.department(),
```

Comment:
Additional request: (this can be in another PR if u want)

Could we choose the RequestGroup name from [this list](https://docs.google.com/document/d/1sC4j4bU41gpVYu9Q_AOO_0EvV4_pVlAbMfWXJshOEMQ/edit)?

Also could we choose the RequestGroup image from this set?
- https://source.unsplash.com/RcgiSN482VI
- https://source.unsplash.com/7ydep8OEvbc
- https://source.unsplash.com/0hiUWSi7jvs

Reply:
Made these changes